src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+     String randomColor = getRandomColor();
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
```

Comment:
Instead of using `String.valueOf(Colors.values()[index])`, use `Colors.values()[index].name()` to get the string representation of the enum constant. This follows the checklist recommendation to use `name()` for getting String representation of enum constants.